bootstrap-salt.sh

```diff
@@ -4457,6 +4469,9 @@ install_opensuse_stable_deps() {
     # shellcheck disable=SC2086
     __zypper_install ${__PACKAGES} || return 1
 
+    # Fix for OpenSUSE 13.2 and 2015.8 - gcc should not be required. Work around until package is fixed by SuSE
+    _EXTRA_PACKAGES="${_EXTRA_PACKAGES} gcc python-devel libgit2-devel" 
```

Comment:
We looked at this with Tom. He said that it looked like the SUSE salt-minion rpm was requiring git event though we don't need it.  I'll have to re-create the problem, but one of the dependencies, (I think git 2) is being required by SUSE. However, it has a dependency that requires a compiler. My plan is to look into it more, submit a bug to SUSE. I've met the packager Olaf when I worked there and wanted to send him a note.  

All I know at this point is that the only way it works is with those packages added. This is just a standard rpm-based installation.